### DISCUSSION TITLE
Printer stopping suddenly during a print with alarm and no indicator.

### DISCUSSION THREAD

**Discussion by NewPrinter** [mriscoc/Ender3V2S1#1372] [Q&A]
I have an Ender 3 V2 using the Professional Firmware for 3D Printers 20240122. I had a print break off of the bed and found I had spaghetti all over. I then printed another and it failed and found a giant wad of plastic all over the hot end. I cleaned that off and went to try my print again.

This was a long print and I kept an eye on it over a couple of hours and it was all going well. In the morning I found that the print was 3/4 done and the print head parked in the "pause print" location. The hot end was clean and a short test print showed it to be clear and not clogged. I thought it might have been a power loss. I tried the print again and about an hour later I heard an alarm on my printer and it was the same earlier state with no indication of what the problem is.

I have researched and found that maybe it might be two issues, either a "wild temp" condition or a bad g-code. I tried another print. I watched it for an hour and all was good. I then went and did a couple of errands and a couple of hours later it was still going. I thought I was in the clear with a g-code issue. I was wrong, it happened again when I went to check on this print. I found it parked and an incomplete print on the bed. I do notice a hot plastic smell now when I print, but I can't find any new on the hot end. I chalked this up to residual from my mess.

Is there any diagnostics on this firmware that I can look at and see what the issue may be? Any advice or helpful things to look at?

**Comment by mriscoc** [comment]
Hi, the printer sends error messages to a terminal like Octoprint, Pronterface, and any other serial console connected to the USB port.

**Reply by NewPrinter** [reply]
Thanks! Unfortunately, I do not have anything connected to the USB port. Being new to printing all I have used is the MicroSD card. For temporary use I guess I could use my laptop connected to the printer. Correct? If so, how and what do I need to set that up?

I use Utilimaker Cura for my slicer.

**Comment by NewPrinter** [comment]
I may have found the problem. I was thinking of what changed. My silicone boot tore during the removal of the glob and I didn't put it on. I thought it only served to keep the head clean and figured a couple of days without would be fine. I did some research and found it also served to keep the temperature stable. I think that might be the issue. I am waiting for the replacement boots to verify.
